Thumbnail zooming

will-moore opened the pull request:
This PR cleans-up the thumbnail zooming logic in webclient.

Instead of using the sizeX and sizeY values to calculate the aspect ratio for every thumbnail and JavaScript sets the width and height on the fly while dragging the thumbnail slider, we now use CSS to do achieve the same effect which is much simpler and might be faster/smoother.

To test:
 - Should see no change in behaviour for thumbnail zooming
 - The table layout icon has a small fix (see screenshots below)

NB: Previously this PR didn't load sizeXYZ values from OMERO until the user clicked on table layout to display them, but this was added complexity for negligible performance gain so it has been removed.

pwalczysko commented:
Tested on 
![Screenshot 2022-12-16 at 11 17 10](https://user-images.githubusercontent.com/2478303/208090631-0a0093ac-8ccb-475d-a5d4-4d43dc17be38.png)

As discussed with @will-moore ^^^, the necessity of the refresh is not a great user experience, and the performance enhancement we saw on merge-ci is less than 10%, with total times well under 1 sec, i.e. not humanly perceptible.
The message wording ``...the data...`` is confusing, I would remove this, but most probably, removing the whole necessity of the refresh is the better option.

pwalczysko reviewed (changes requested):
some suggestions in https://github.com/ome/omero-web/pull/411#issuecomment-1354609284

will-moore commented:
Thanks @pwalczysko: fixed and re-deployed (and updated description).
Now, the only visible change in behaviour is the tiny fix to layout buttons. When on table layout:

Before:

![Screenshot 2022-12-16 at 13 15 37](https://user-images.githubusercontent.com/900055/208106727-ada78f91-f76b-4565-992b-3904f324e386.png)

After:

![Screenshot 2022-12-16 at 13 19 48](https://user-images.githubusercontent.com/900055/208107205-2a746c56-0513-4f18-978d-e62aaef0e734.png)

jburel commented:
Excluding ahead of the dependencies upgrade work

--exclude

jburel commented:
``exclude`` label removed

pwalczysko commented:
lgtm
